internal/characters/faruzan/asc.go

```diff
@@ -0,0 +1,53 @@
+package faruzan
+
+import (
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/glog"
+)
+
+const (
+	a4Key    = "faruzan-a4"
+	a4ICDKey = "faruzan-a4-icd"
+)
+
```

Comment:
nit: Missing a4 description? You added ability descriptions for all the other functions.